### DISCUSSION TITLE
The EXE file is difficult to find

### DISCUSSION THREAD

**Discussion by GabrielOnDelphi** [valinet/ExplorerPatcher#2252] [General]
Hi
The precompiled EXE file is difficult to find.
Maybe you can link to the download page, from the home page.
https://github.com/valinet/ExplorerPatcher/releases/
That's it.

Mersi pt acest super program. You have done what the whole Microsoft could not do!

**Comment by Imold** [comment]
It already is.
![Screenshot 2023-10-07 1208372](https://github.com/valinet/ExplorerPatcher/assets/3978545/c42fc733-a42c-4748-b27d-38fa8ddccdf7)

**Comment by Poopooracoocoo** [comment]
I think it's like that intentionally. The readme has a link to the wiki and only then has a link to the downloads. You have to at least have a glance of the feature summary before downloading. And besides the readme, it's already easy to access the releases.
